src/coord/radar/RadarModel.ts

```diff
@@ -151,11 +151,9 @@ class RadarModel extends ComponentModel<RadarOption> implements CoordinateSystem
                     innerIndicatorOpt.name, innerIndicatorOpt
                 );
             }
-            const model = zrUtil.extend(
-                new Model(innerIndicatorOpt, null, this.ecModel),
-                AxisModelCommonMixin.prototype
-            ) as AxisBaseModel<InnerIndicatorAxisOption>;
 
+            mixinAxisModelCommonMethods(Model);
+            const model = new Model(innerIndicatorOpt, null, this.ecModel) as AxisBaseModel<InnerIndicatorAxisOption>;
```

Comment:
Don't mixin the methods to the Model#prototype. It will affect all the models.

Reply:
@pissang Please take a look